olclient/openlibrary.py

```diff
@@ -1043,7 +1049,7 @@ def __init__(self, key, title=u"", subtitle=None, subject=None,
             # XXX test that during the zip, author_name and author_key
             # correspond to each other one-to-one, in order
             self.authors = [
-                common.Author(name=name, identifiers={u'olid': [author_olid]})
+                self.OL.Author(olid=author_olid, name=name)
```

Comment:
I think this should be `Openlibrary.Author(olid=author_olid, name=name)`  (after you remove the `OL` line further up)

Reply:
@hornc and I didn't see a need for an Author object creation here as the `Results.Document` object is just for representation of data in data retrievals.
This is handled in commit:[
fb9c399 ](https://github.com/internetarchive/openlibrary-client/pull/133/commits/fb9c3997af869c8fac59242a759e42fec2ba226f)